crates/biome_js_analyze/tests/specs/nursery/noUnusedImports/valid.jsx

```diff
@@ -0,0 +1 @@
+import { React } from "react"
```

Comment:
`import { React } from "react"` is not a valid react import because it need to be a default import

Here some valid imports:

```js
import React from 'react';
import { default as React  } from 'react';
import * as React from 'react';
```

Reply:
Similar for `ReactDOM` with `react-dom`?

Reply:
Yes